boards/arm/stm32l476g_disco/pinmux.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2017 Arthur Sfez
+ *
+ * Based on stm32f412g_disco:
```

Comment:
Nice, but I think you could remove this. All pinmux files are copied from one another.